AUTH.md

```diff
@@ -16,7 +16,34 @@ Whenever a request is run towards a WITSML server, the backend will decrypt the
 This is how the flow is when a user has selected a server and will need to authenticate against it. After this is done, a fresh list of wells is fetched.  
 
 
-<img src="./credentials-flow.svg">
+```mermaid
+sequenceDiagram
+    Note left of Frontend: User Select  WITSML <br> server for the first <br> time in current session
+    Frontend->>+Frontend: No credentials exist,<br/> display credentials <br/> dialog
+    Note left of Frontend: Enter and submit <br> credentials for <br> WITSML server
+    Frontend->>+Api: Verify credentials using <br/> basic auth /api/credentials/authorize
+    Api->>-Witsml Server: Verify credentials by <br/> querying for server capabilities <br/> using basic auth
+    activate Witsml Server
+    Witsml Server ->>- Api: Ok / No access 
+    activate Api
+    Api->>-Frontend: If not valid, return error and <br> display credentials dialog
+    activate Api
+    Api->>+Api: If valid, encrypt password <br> and store it in memory <br> using DataProtection library
```

Comment:
Should the activation be stacked for actions that loop back to the same actor? Disabling stacked activation could be achieved by replacing "Api->>+Api" with "Api->>Api" on line 31 and "Api->>-Frontend" with "Api->>Frontend" on line 32, as an example. This would give us single activation per actor at a time just like in the original diagram.

Reply:
Good tip. Implemented now